Week-4/Async_JS/Exchangerates/script.js

```diff
@@ -0,0 +1,16 @@
+async function getExchangeRate(currencyCode) {
+  try {
+    const response = await fetch(`https://api.exchangerate.host/latest`);
+    const output = await response.json();
+    const rates = output.rates;
+    if (rates.hasOwnProperty(currencyCode)) {
+      console.log(rates[currencyCode].toPrecision(5));
+      return rates[currencyCode].toPrecision(5);
+    } else {
+      return null;
+    }
+    //return output.rates.currencyCode;
+  } catch (e) {
+    return 'Error while fetching';
+  }
+}
```

Comment:
where are you calling this function?

Reply:
in the console of the html file attached to it

Reply:
i don't see it in html file though. You can invoke the function after definition and on running html it should implicitly call the function.